tests/infra/node.py

```diff
@@ -255,7 +256,24 @@ def get_ledger(self):
         return self.remote.get_ledger()
 
     def get_snapshots(self):
-        return self.remote.get_snapshots()
+        # Wait for all available snapshot files to be committed before
+        # copying snapshot directory
+        def wait_for_snapshots_to_be_committed(src_dir, list_src_dir_func, timeout=3):
+            end_time = time.time() + timeout
+            committed = True
+            while time.time() < end_time:
+                committed = True
+                for f in list_src_dir_func(src_dir):
+                    committed &= f.endswith(".committed")
+                if committed:
+                    break
+                time.sleep(0.1)
+            if not committed:
+                raise RuntimeError(
+                    f"All snapshots were not committed after {timeout}s in {src_dir}"
```

Comment:
Should that read "Not all snapshots..."? Printing the set of uncommitted ones may be quite helpful too.